include/alpaka/mem/view/Traits.hpp

```diff
@@ -115,8 +115,9 @@ namespace alpaka
                         -> idx::Idx<TView>
                         {
                             return
-                                extent::getExtent<dim::Dim<TView>::value - 1u>(view)
-                                * sizeof(elem::Elem<TView>);
+                                static_cast<idx::Idx<TView>>(
+                                    static_cast<unsigned int>(extent::getExtent<static_cast<unsigned int>(dim::Dim<TView>::value) - 1u>(view))
```

Comment:
What exactly made it necessary to cast the non-type-template parameters here?

Reply:
I got warnings there and it was either cast `1u` to `Idx` or cast the rest to `unsigned int`, I went with the latter. I think there are cases where `idx::Idx<TView>` is signed, maybe creating warnings because of the unsigned value returned by `sizeof`.

Reply:
The `getExtent<static_cast<unsigned int>` seems to be valid.
The outer `static_cast<idx::Idx<TView>>` makes it explicit what the `return` will do anyways.
However, the `static_cast<unsigned int>(extent::getExtent<` is dangerous because we should not change / clip the values returned by `getExtent`.

Reply:
Maybe test causing this issue were changed in the meantime, if I revert this hunk, I can now build without warnings.

Should I remove this change or just reduce it to the innermost cast to `unsigned int`?

Reply:
I would completely remove it